.vscode/settings.json

```diff
@@ -1,5 +1,11 @@
 {
   "cSpell.enabled": true,
   "editor.formatOnSave": true,
-  "rust-analyzer.cargo.features": "all"
+  "rust-analyzer.cargo.features": "all",
+  "[powershell]": {
+    "editor.defaultFormatter": "ms-vscode.powershell",
+  },
+  // the yaml formatter will use complex key notation to wrap long lines
+  // allow really long lines to prevent that.
```

Comment:
I appreciate VSCode supports comments, but GitHub does not and this immediately raises red flags that turn out to be red herrings in code reviews:

![image](https://github.com/user-attachments/assets/5088c0e9-2009-41d3-b929-253c69451436)

If the comments don't add any real substance I'd recommend removing them.

Reply:
comment removed